### PR TITLE
Add GetIDs method

### PR DESCRIPTION
**Pull request by tzdybal**
## Overview

`GetIDs` method return all IDs available at given height. This method combined with `Get` provides functionality of `GetAll`, but at the same time is more flexible. Getting contents of all blobs might be the heavy and unnecessary. 

Resolves https://github.com/rollkit/go-da/issues/10

## Checklist

- [x] New and updated code has appropriate documentation
- [x] New and updated code has new and/or updated testing
- [x] Required CI checks are passing
- [x] Visual proof for any user facing features like CLI or documentation updates
- [x] Linked issues closed with keywords

### CODE REVIEW

**Comment by tzdybal** [comment]
@MSevey, I have some questions:
1. for some reason test & lint jobs are not executed - anything missing in the repo?
2. team mention is not replaced by individual team members in Reviewers - how to fix this?

**Comment by MSevey** [comment]
> @MSevey, I have some questions:
> 
> 1. for some reason test & lint jobs are not executed - anything missing in the repo?

Error in the workflow file https://github.com/rollkit/go-da/actions/runs/6379741787
we don't have proto in this repo?

**Comment by MSevey** [comment]
> team mention is not replaced by individual team members in Reviewers - how to fix this?

It looks like you manually requested the review? If so, then it doesn't automatically assign someone.
The automatic review assignment comes from the housekeeping workflow which runs when you open the PR. That will pick 3 reviewers for the PR from the team. 

It looks like the auto review request hit some ignore case: https://github.com/rollkit/go-da/actions/runs/6379741784/job/17313941628?pr=11#step:2:8

Did you initially open the PR as a draft? There might be an edge case with ignoring drafts but only running the workflow on open PRs.

File: da_test.go [hunk 1]
```diff
@@ -90,3 +94,36 @@ func CheckErrors(t *testing.T, da da.DA) {
...
+	assert.NoError(t, err)
+	assert.Len(t, ids, len(msgs))
+	assert.Len(t, proofs, len(msgs))
+
+	found := false
+	end := time.Now().Add(1 * time.Second)
+	for i := uint64(1); !found && !time.Now().After(end); i++ {
+		ret, err := da.GetIDs(i)
+		if err != nil {
+			break
```
Comment (MSevey) [inline da_test.go:111]: nit:
I think there is more to learn by making this `t.Error("unable to get ids:",err)`
The test would still fail, and we might see something in certain heights failing.

**Review by MSevey — DISMISSED** [review]
just a small nit.

**Review by MSevey — APPROVED** [review]

**Review by nashqueue — APPROVED** [review]

**Pull request merged**